eks_fargate/datadog_checks/eks_fargate/eks_fargate.py

```diff
@@ -0,0 +1,29 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.base import AgentCheck
+
+
+class EksFargateCheck(AgentCheck):
+    """
+    Generate a heartbeat for Amazon EKS on AWS Fargate workloads
+    """
+
+    def __init__(self, name, init_config, instances):
+        super(EksFargateCheck, self).__init__(
+            name, init_config, instances,
+        )
+
+        self.virtual_node = os.getenv("DD_KUBERNETES_KUBELET_NODENAME")
+        self.pod_name = os.getenv("HOSTNAME")
+
+    def check(self, instance):
+
+        # Only submit the heartbeat metric for fargate virtual nodes.
+        if "fargate" in self.virtual_node:
+            tags = []
```

Comment:
looks like tags could be defined directly in the init then, removing `self.virtual_node` & `self.pod_name` and keeping only `self.tags`? 🤔

Reply:
done

Reply:
Was this reverted?

Reply:
Yeah I ended up reverting, we still need virtual_node in the self to evaluate if we are in fargate mode so the benefit did not seem that significant.

Reply:
Not really? set a boolean